docs/typelibrary/io/QL.md

```
@@ -1,3 +1,56 @@
-### QL - Output LWORD (64 Bit)
+# QL
 
 ![](https://user-images.githubusercontent.com/69573151/210781586-ad33d398-c369-4e16-b79e-20ed939fcdd6.png)
+
+* * * * * * * * * *
+
+## Einleitung
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)